.github/workflows/dev-build.yml

```diff
@@ -11,6 +11,24 @@ on:
       - .gitignore
 
 jobs:
+  get-previous-tag:
```

Comment:
I don't think that we need to check changes for the dev build. It might become hard to find the version of grpc in that case because it is not published every time the build is executed

Reply:
Sorry, I miss understood your first comment)
I will remove this job dev build. I added this job for testing